packages/app-store/templates/audit-log-implementation/components/AppSettingsInterface.tsx

```diff
@@ -0,0 +1,132 @@
+import { zodResolver } from "@hookform/resolvers/zod";
```

Comment:
This component will house all settings, and features for any audit log implementation. By default it exposes a CredentialForm and the EventToggles interface. Users can easily extend this to include things like a logger-viewer ( in BoxyHQs case) Default components can also be removed